domains/arca.json

```
@@ -0,0 +1,11 @@
+{
+    "repo": "https://github.com/anibalCApaza/arcadev.github.io",
+    "owner": {
+      "username": "arca",
+      "email": "arcacorini@gmail.com"
+    },
+    "record": {
+      "CNAME": "https://anibalcapaza.github.io/arcadev.github.io/"
```

```suggestion
      "CNAME": "anibalcapaza.github.io"
```
CNAME records cannot contain protocols (`https://`) or slashes (`/`).